### DISCUSSION TITLE
When will Django 3.2 be supported?

### DISCUSSION THREAD

**Discussion by fabtjar** [encode/django-rest-framework#7993] [General] [answered]
The requirements doesn't show Django 3.2 yet: https://www.django-rest-framework.org/#requirements

If anyone is working on getting 3.2 support do you know when this will be out?

**Comment by carltongibson** [comment, marked as answer]
It's already supported. See https://github.com/encode/django-rest-framework/discussions/7944

Duplicate of #7944

**Reply by FinnGu** [reply]
@carltongibson I could update the `README.md` if it is officially supported now. I too thought, it was still in testing.

**Reply by lovelydinosaur** [reply]
That would be great, thanks! We're really bottlenecked on maintainer time at the moment, and clearly need to do a better job of getting the project into a community managed place. (So that I'm providing some oversight and direction, but am not necessarily the bottleneck on items being progressed.)

**Reply by FinnGu** [reply]
Cool, I'm glad I can contribute a little towards this project! Have been using it for over two years now and it really is awesome!

https://github.com/encode/django-rest-framework/pull/8037
